validator/storage_bucket_policy_only_test.rego

```diff
@@ -16,22 +16,28 @@
 
 package templates.gcp.GCPStorageBucketPolicyOnlyConstraintV1
 
+import data.test.fixtures.storage_bucket_policy_only.assets as fixture_buckets
+
 all_violations[violation] {
-	resource := data.test.fixtures.storage_bucket_policy_only.assets[_]
 	constraint := data.test.fixtures.storage_bucket_policy_only.constraints.require_bucket_policy_only
 
-	issues := deny with input.asset as resource
+	issues := deny with input.asset as fixture_buckets[_]
 		 with input.constraint as constraint
 
 	violation := issues[_]
 }
 
 # Confirm total violations count
 test_storage_bucket_policy_only_violations_count {
-	count(all_violations) == 2
+	count(all_violations) == count(fixture_buckets) - 1
```

Comment:
optional: to be more explicit, we can verify that the "good" bucket is not in the violations list. Then we can do away with this brittle counting business.

Reply:
restructured tests to mirror functions in iam_audit_log_test.rego, removed the need to count.